Authfix

mahtai opened the pull request:
Fixing the unit authorization problem, so that the creator of a unit is now attached to the unit itself. Also fixed some view issues on the user and unit sides.

mahtai committed:
Unit creator added to unit

mahtai committed:
Street address replaced by unit address

mahtai committed:
Unit creator added to unit

mzagaja commented:
I wonder if the access denied is because I did not start with a fresh user. This mostly works but the view of the unit seems to be missing its occupants and there are two random non-link hover thingies at the bottom whose purpose is unclear:

<img width="413" alt="screenshot 2016-05-02 21 43 31" src="https://cloud.githubusercontent.com/assets/565647/14972878/0830f93a-10af-11e6-9e77-b3ad2574399e.png">

mahtai committed:
Got rid of occupants table

mahtai commented:
The occupants table should show the name of the occupants who are users. But this may not be the best method to indicate who is in the unit, as not all occupants may be users and so the table will be missing some things. Also users who haven't filled in a name won't display properly. I'll figure this out later.

mahtai committed:
user view tweak

mahtai committed:
update authfix branch

ranch 'origin' into authfix

# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

mahtai committed:
Merging user view

mahtai commented:
Huh. One of the recent commits broke the unit-user_building assignment.
